package.json

```diff
@@ -91,7 +91,7 @@
     "core-js": "3.6.5",
     "flexboxgrid": "6.3.1",
     "focus-trap": "5.1.0",
-    "fundamental-styles": "0.9.0",
+    "fundamental-styles": "0.10.0",
```

Comment:
Have we checked to see if upgrading `fundamental-styles` is OK with the fundamental-ngx team? Sometimes upgrading introduces some unexpected changes in other parts of the code library, and we need to make sure nothing breaks.

Reply:
Checking with them, it is. causing build break, we are using styles of 0.10 for the list header. for now. Will act accordingly once we get response from them

Reply:
Updated with latest version issue is resolved now.